spec/grape-active_model_serializers/formatter_spec.rb

```diff
@@ -65,5 +84,9 @@ def endpoint.default_serializer_options
       expect(subject.instance_variable_get('@only')).to eq([:only])
       expect(subject.instance_variable_get('@except')).to eq([:except])
     end
+
+    it 'should read serializer options like "root"' do
+      expect(described_class.build_options_from_endpoint(endpoint).keys).to include :root
```

Comment:
Yes, I took the 'root' option as an example here, but the same goes for the 'serialiser' and 'each_serializer' options. In the current implementation these options get lost.... In this spec the other options could be tested for presence as well